app/modules/asset_groups/tasks.py

```diff
@@ -82,7 +82,7 @@ def git_push(asset_group_guid):
 def sage_detection(asset_group_sighting_guid, model):
     from .models import AssetGroupSighting
 
-    asset_group_sighting = AssetGroupSighting.query.find(asset_group_sighting_guid)
+    asset_group_sighting = AssetGroupSighting.query.get(asset_group_sighting_guid)
```

Comment:
what's the benefit of this?

Reply:
`AssetGroupSighting.query.find` doesn't exist, so I changed it to `AssetGroupSighting.query.get`